aten/src/ATen/KinetoPluginAPI.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <c10/core/DeviceType.h>
+#include <c10/macros/Macros.h>
+
+#include <libkineto.h>
+
+namespace c10 {
+namespace kineto_plugin {
+
+using PluginFactory = libkineto::ChildActivityProfilerFactory;
+void C10_API insertKinetoPluginRegister(DeviceType t, PluginFactory f);
+void C10_API registerKinetoPluginProfiler();
```

Comment:
Should be not C10_API anymore?

Reply:
We should not expose registerKinetoPluginProfiler to user?

Reply:
> Should be not C10_API anymore?

yes

Reply:
> We should not expose registerKinetoPluginProfiler to user?

this api is not exposed to users.